### PR TITLE
fix SRCREV_FORMAT errors

### PR DESCRIPTION
**Pull request by awe-1989**

### CODE REVIEW

**Comment by akuster** [comment]
the commits from awe-1989 are missing signed-off-by

File: meta-oe/recipes-graphics/spir/spirv-tools_git.bb [hunk 1]
```diff
@@ -21,6 +21,7 @@ SRCREV_spirv-headers = "af64a9e826bf5bb5fcd2434dd71be1e41e922563"
 SRCREV_effcee = "cd25ec17e9382f99a895b9ef53ff3c277464d07d"
 SRCREV_re2 = "5bd613749fd530b576b890283bfb6bc6ea6246cb"
 SRCREV_googletest = "f2fb48c3b3d79a75a88a99fba6576b25d42ec528"
+SRCREV_FORMAT ="ttols_headers_effcee_re2_googletest"
```
Comment (kraj) [inline meta-oe/recipes-graphics/spir/spirv-tools_git.bb:24]: there is a typo s/ttools/spirv-tools/ and `headers` should be `spirv-headers`

**Review by kraj — CHANGES_REQUESTED** [review]

File: meta-networking/recipes-support/drbd/drbd-utils_9.12.0.bb [hunk 2]
```diff
@@ -15,6 +15,7 @@ SRC_URI = "git://github.com/LINBIT/drbd-utils;name=drbd-utils \
 # v9.12.0
 SRCREV_drbd-utils = "91629a4cce49ca0d4f917fe0bffa25cfe8db3052"
 SRCREV_drbd-headers = "233006b4d26cf319638be0ef6d16ec7dee287b66"
+SRCREV_FORMAT = "utils_headers"
```
Comment (kraj) [inline meta-networking/recipes-support/drbd/drbd-utils_9.12.0.bb:18]: this should be drbd-utils_drbd-headers

**Comment by awe-1989** [comment]
Thanks for the comments. I pushed a new patch set.

**Review by kraj — APPROVED** [review]
this looks ok to me. I see that you proposed it for dunfell. I would like you to first ensure that all these changes are already present in one way or other in master branch as well. Then we can apply it for dunfell.

**Comment by awe-1989** [comment]
> this looks ok to me. I see that you proposed it for dunfell. I would like you to first ensure that all these changes are already present in one way or other in master branch as well. Then we can apply it for dunfell.

meta-networking/recipes-support/drbd/drbd-utils_9.12.0.bb was changed to drbd-utils_9.13.1.bb and fixed in bd499150d29a7c62d706f85ca5d879b2db772192

meta-oe/recipes-devtools/grpc/grpc_1.24.3.bb was changed to grpc_1.35.0.bb. during this time was switched to a single source. 15cff67fd6cdb34e3621368fe9ce94a98356f27a

meta-oe/recipes-graphics/spir/spirv-tools_git.bb was moved to openembedded-core with 8a5d7f28a02a18cd26749f7b97c1e85128fda720. There it has only one source in the master branch.

Thus, these problems should no longer occur in the master branch.

**Comment by akuster** [comment]
pulled changes in for testing

**Comment by akuster** [comment]
some of these also pulled in for hardknott

**Comment by akuster** [comment]
merged
https://git.openembedded.org/meta-openembedded/commit/?h=dunfell&id=f8a2a7e41ac86371dd2b2b95c94d29c335f8676d

**Comment by akuster** [comment]
thanks

**Pull request closed without merging**